pio-scripts/inject_syslog_ui.py

```diff
@@ -0,0 +1,196 @@
+# pio-scripts/inject_syslog_ui.py
+
+"""
+PlatformIO build script to conditionally inject Syslog UI elements into the settings HTML file.
+
+This script:
+1. Injects Syslog UI elements when WLED_ENABLE_SYSLOG is defined in build flags
+2. Restores the original HTML file after build completion
+3. Tracks state between builds to force UI rebuilds when necessary
+"""
+
+import os, re, shutil
+from SCons.Script import Import
+
+Import("env")
+
+# detect full vs. partial compile
+is_full_build = env.get("PIOENV") is not None
+
+# Track the state between builds
+def get_previous_syslog_state(project_dir):
+    state_file = os.path.join(project_dir, "wled00/data/.syslog_state")
+    if os.path.exists(state_file):
+        with open(state_file, 'r') as f:
+            return f.read().strip() == "1"
+    return None  # None means no previous state recorded
+
+def set_syslog_state(project_dir, enabled):
+    state_file = os.path.join(project_dir, "wled00/data/.syslog_state")
+    with open(state_file, 'w') as f:
+        f.write("1" if enabled else "0")
+
+# This is the HTML we want to inject
+SYSLOG_HTML = """
+<h3>Syslog</h3>
+<div id="Syslog">
+  Enable Syslog: <input type="checkbox" name="SL_en"><br>
+  Host: <input type="text" name="SL_host" maxlength="32"><br>
+  Port: <input type="number" name="SL_port" min="1" max="65535" value="%SL_port%"><br>
+
+  <!-- These UI elements are commented out but preserved for potential future use -->

```

Comment:
remove or if we must keep, have as comments in the python not redundant code in the html